File: src/tests/stuff/moretest/sorting.c
#include <stdio.h>
int main() {
    printf("sorting\n[");
    int arr[150];
    for (int i = 0; i < 150; i++) arr[i] = 150 - i;
    for (int i = 0; i < 150; i++)
        for (int j = i + 1; j < 150; j++)
            if (arr[j] < arr[i]) { int t = arr[i]; arr[i] = arr[j]; arr[j] = t; }
    for (int i = 0; i < 150; i++) {
        if (i > 0) printf(", ");
        printf("%d", arr[i]);
    }
    printf("]\n");
    return 0;
}

File: src/tests/stuff/moretest/sorting.rs
fn main() {
    println!("sorting");
    let mut arr: [i32; 150] = [0; 150];
    for i in 0..150 { arr[i] = 150 - i as i32; }
    for i in 0..150 {
        for j in i+1..150 {
            if arr[j] < arr[i] { let t = arr[i]; arr[i] = arr[j]; arr[j] = t; }
        }
    }
    println!("{:?}", arr.to_vec());
}
